pilot/pkg/xds/ads.go

```diff
@@ -681,10 +681,7 @@ func (s *DiscoveryServer) shouldProcessRequest(proxy *model.Proxy, req *discover
 // The delta protocol changes the request, adding unsubscribe/subscribe instead of sending full
 // list of resources. On the response it adds 'removed resources' and sends changes for everything.
 func (s *DiscoveryServer) DeltaAggregatedResources(stream discovery.AggregatedDiscoveryService_DeltaAggregatedResourcesServer) error {
-	if features.DeltaXds {
```

Comment:
why do we want to remove this? I think we still want to return not implemented if we don't fully support/people don't actively enable the flag.

Reply:
I don't see why we need to block it on control plane and data plane side. If data plane is not configured to use delta, it would never be used. The only reason I guess would be if we think allowing clients to connect to delta introduces risk to the control plane, but I am not sure that is too large of a concern.

The reason I did it is that we would need a custom deployment of Istiod to test delta as is; with this removed we can just deploy a single workload using delta

Reply:
fair enough